apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/ScheduleSelector.tsx

```diff
@@ -20,43 +25,45 @@ class ScheduleSelector extends PureComponent<ScheduleSelectorProps, ScheduleSele
         scheduleIndices: this.props.scheduleIndices,
     };
 
-    handleChange = (dayIndex: number) => (event: React.ChangeEvent<HTMLInputElement>) => {
-        const checked = event.target.checked;
+    handleChange = (event: SelectChangeEvent<typeof this.state.scheduleIndices>) => {
+        const value = event.target.value as number[];
 
-        this.setState(
-            (prevState) => {
-                const newScheduleIndices = checked
-                    ? [...prevState.scheduleIndices, dayIndex]
-                    : prevState.scheduleIndices.filter((scheduleIndex) => {
-                          return scheduleIndex !== dayIndex;
-                      });
-
-                return { scheduleIndices: newScheduleIndices };
-            },
-            () => this.props.onSelectScheduleIndices(this.state.scheduleIndices)
-        );
+        this.setState({ scheduleIndices: value }, () => this.props.onSelectScheduleIndices(this.state.scheduleIndices));
     };
 
     render() {
         return (
-            <FormGroup row>
-                {this.props.scheduleNames.map((name, index) => {
-                    return (
-                        <FormControlLabel
-                            control={
-                                <Checkbox
-                                    checked={this.state.scheduleIndices.includes(index)}
-                                    onChange={this.handleChange(index)}
-                                    value={index + 1}
-                                    color="primary"
-                                />
-                            }
-                            label={name}
-                            key={name}
-                        />
-                    );
-                })}
-            </FormGroup>
+            <FormControl style={{ marginTop: 10, maxWidth: 400 }} fullWidth variant="outlined">
+                <InputLabel id="schedule-select-label" htmlFor="select-multiple-chip">
+                    Select schedules
+                </InputLabel>
+                <Select
+                    labelId="schedule-select-label"
+                    id="schedule-select"
+                    size="small"
+                    style={{ marginTop: 2 }}
```

Comment:
```suggestion
                    style={{ marginTop: 2 }}
```
nit: i think we can do without the margin top here